docs/HowToUsePeripheral.md

```diff
@@ -501,9 +516,222 @@ static const struct i2s_ops_s g_i2sops = {
+	 * the given parameter (frequency, duty, or optionally pulse count)
+	 */
+
+	CODE int (*start)(FAR struct pwm_lowerhalf_s *dev, FAR const struct pwm_info_s *info);
+
+	/* Stop the pulsed output and reset the timer resources */
+
+	CODE int (*stop)(FAR struct pwm_lowerhalf_s *dev);
+
+	/* Lower-half logic may support platform-specific ioctl commands */
+
+	CODE int (*ioctl)(FAR struct pwm_lowerhalf_s *dev, int cmd, unsigned long arg);
+};
+
+/****************************************************************************
+ * Name: pwm_setup
+ *
+ * Description:
+ *   This method is called when the driver is opened.  The lower half driver
+ *   should configure and initialize the device so that it is ready for use.
+ *   It should not, however, output pulses until the start method is called.
+ *
+ * Input parameters:
+ *   dev - A reference to the lower half PWM driver state structure
+ *
+ * Returned Value:
+ *   Zero on success; a negated errno value on failure
+ *
+ ****************************************************************************/
+static int pwm_setup(FAR struct pwm_lowerhalf_s *dev);
+
+
+struct pwm_info_s {
+	uint32_t frequency;			/* Frequency of the pulse train */
+	ub16_t duty;				/* Duty of the pulse train, "1"-to-"0" duration.
+};
+
+/****************************************************************************
+ * Name: pwm_start
+ *
+ * Description:
+ *   (Re-)initialize the timer resources and start the pulsed output
+ *
+ * Input parameters:
+ *   dev - A reference to the lower half PWM driver state structure
+ *   info - A reference to the characteristics of the pulsed output
+ *
+ * Returned Value:
+ *   Zero on success; a negated errno value on failure
+ *
+ ****************************************************************************/
+static int pwm_start(FAR struct pwm_lowerhalf_s *dev,
+						   FAR const struct pwm_info_s *info);
+
+/****************************************************************************
+ * Name: pwm_stop
+ *
+ * Description:
+ *   Stop the pulsed output and reset the timer resources
+ *
+ * Input parameters:
+ *   dev - A reference to the lower half PWM driver state structure
+ *
+ * Returned Value:
+ *   Zero on success; a negated errno value on failure
+ *
+ * Assumptions:
+ *   This function is called to stop the pulsed output at anytime.  This
+ *   method is also called from the timer interrupt handler when a repetition
+ *   count expires... automatically stopping the timer.
+ *
+ ****************************************************************************/
+static int pwm_stop(FAR struct pwm_lowerhalf_s *dev);
+```
+
+>This function makes 'dev' to stop.
+But it still has its own resources so that it can be re-started.
+
+
+```c
+
+/****************************************************************************
+ * Name: pwm_shutdown
+ *
+ * Description:
+ *   This method is called when the driver is closed.  The lower half driver
```

Comment:
could you remove the whit space?
"closed. This"

Reply:
removed.